src/aatoolbox/config/countries/bgd.yaml

```diff
@@ -0,0 +1,6 @@
+iso3: bgd
+codab:
+  hdx_address: administrative-boundaries-of-bangladesh-as-of-2015
```

Comment:
not specific for this PR but just making a note that the address and name might change at some point. Could we see with the HDX team if we can get an update on that or smth?

Reply:
Yeah that's tricky, for this part of the pipeline we could have it setup to run once a day or something which would serve as a check

Reply:
Yes that could work. Also think it is good to ask HDX team what they would recommend. Lets leave it for now and make an issue?

Reply:
Created #30